.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "rust-analyzer.linkedProjects": [
+        //"./extendr-api/Cargo.toml",
+        "./extendr-macros/Cargo.toml",
+        //"./src/rust/Cargo.toml",
+        //"py-polars"
+    ],
+}
```

Comment:
You checked in `settings.json` specific to your environment. I suggest not including it (at least in the current form).